internal/querycoordv2/observers/collection_observer.go

```diff
@@ -34,9 +34,11 @@ import (
 type CollectionObserver struct {
 	stopCh chan struct{}
 
-	dist      *meta.DistributionManager
-	meta      *meta.Meta
-	targetMgr *meta.TargetManager
+	dist                *meta.DistributionManager
+	meta                *meta.Meta
+	targetMgr           *meta.TargetManager
+	collectionLoadedMap map[int64]int
```

Comment:
`collectionLoadedMap` -> `collectionLoadedCount`, The type has implied it's a map